libnemo-private/nemo-file-undo-operations.c

```diff
@@ -1139,8 +1138,12 @@ trash_retrieve_files_to_restore_thread (GSimpleAsyncResult *res,
 				orig_trash_time = GPOINTER_TO_SIZE (lookupvalue);
 				time_string = g_file_info_get_attribute_string (info, G_FILE_ATTRIBUTE_TRASH_DELETION_DATE);
 				if (time_string != NULL) {
-					g_time_val_from_iso8601 (time_string, &timeval);
-					trash_time = timeval.tv_sec;
+       					/* For the final trash time, use a cast instead of a
+		 			   g_date_time_to_unix. All this does is try to make GLib
+		  		    	   process a NULL timezone, and it floods the logs with
+				 	   assertion failures. */
+					GDateTime *time = g_date_time_new_from_iso8601 (time_string, NULL);
```

Comment:
This returns null because time_string contains no timezone - if it does, THEN you can leave default_tz null.